engines/dfc_provider/config/routes.rb

```diff
@@ -5,7 +5,7 @@
   resources :enterprises, only: [:show] do
     resources :catalog_items, only: [:index, :show, :update]
     resources :offers, only: [:show, :update]
-    resources :orders, only: [:create]
+    resources :orders, only: [:show, :create]
```

Comment:
A route (URL) has been added for the order so that it can have a semanticId, but we haven't implemented the show action yet.

Reply:
We discussed nesting in the enterprise scope vs not nesting the URL. I preferred the shorter version but I now see a comment from Maxime that favours the nesting. All endpoints are related to an enterprise:

* https://github.com/datafoodconsortium/ontology/issues/150#issuecomment-2776147424